<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatVec3Widget</class>
 <widget class="QWidget" name="FloatVec3Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FloatVec3Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="xData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="yData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="zData"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
